packages/widget/src/widget.ts

```diff
@@ -1,14 +1,37 @@
 import type { HTMLTemplateResult } from 'lit';
 import { LitElement, html } from 'lit';
-import { customElement } from 'lit/decorators.js';
+import { customElement, property } from 'lit/decorators.js';
 import './components/widget-app';
+import type { ethers } from 'ethers';
+import type { ApiPromise } from '@polkadot/api';
 import { styles } from './styles';
 import { Network } from './controllers';
+import type { ISygmaProtocolWidget } from './interfaces';
 
 @customElement('sygmaprotocol-widget')
 class SygmaProtocolWidget extends LitElement {
```

Comment:
I think we also need:
- "show" od "disable" attribute.
- "whitelistedResource" property to enable dapp to lock widget to only transfer their token. Maybe in https://chainagnostic.org/CAIPs/caip-19

Reply:
Would it make sense to also define at the interface level, style options for the widget to pass as properties? We discussed about this with @MakMuftic but we didin't arrive to any agreement and I guess it makes sense, since we are defining the interfaces here.

Reply:
yes!